packages/dnb-eufemia/src/components/date-picker/style/dnb-date-picker.scss

```
@@ -523,3 +523,66 @@
     opacity: 0;
   }
 }
+
+.dnb-date-picker__portal {
+  // make variables available inside the portal
+  --date-picker-input-height: 2rem;
+  --date-picker-day-width: 2rem;
+  --date-picker-day-horizontal-spacing: 4px;
+
+  position: absolute;
+  left: 0;
+  z-index: 1000;
```

```suggestion
  z-index: 1000;
```

I think we have documented the z-index number for other components. Have we?

---

Hmm, not sure 🤔 Will see if I can find something 👍